contracts/coordinator/contracts/src/MixinCoordinatorApprovalVerifier.sol

```diff
@@ -164,11 +173,13 @@ contract MixinCoordinatorApprovalVerifier is
             });
 
             // Ensure approval has not expired
-            require(
-                // solhint-disable-next-line not-rely-on-time
-                currentApprovalExpirationTimeSeconds > block.timestamp,
-                "APPROVAL_EXPIRED"
-            );
+            // solhint-disable-next-line not-rely-on-time
+            if (currentApprovalExpirationTimeSeconds <= block.timestamp) {
```

Comment:
Now that 0x transactions have an expiration field, we should probably remove this entirely. Feels redundant to have an expiration in both the approval and the transaction.